.github/workflows/build-and-test.yml

```diff
@@ -487,6 +487,62 @@ jobs:
             throw "Failed to uninstall fluentdwinsvc."
           }
 
+  windows-zeroconfig-e2e-test:
+    runs-on: windows-2022
```

Comment:
Is there any value in also testing on `windows-2019`?

Reply:
In principle no: there shouldn't be any difference affecting the versions shipped with the .NET instrumentation, nor the app itself. There is always a possibility of some difference, but, it is extremely unlikely in this case.